src/igtcloud/client/tools/upload_project.py

```diff
@@ -32,6 +34,32 @@ def upload_project(local_folder: str, project_name: str, institute_name: str = N
     if submit:
         _password = getpass("For electronic record state it is required to reenter the password")
 
+    if category.lower() == "annotation":
+        # Annotation file upload
+        logger.info("Annotation File Uploading...")
+        if os.path.isdir(local_folder):
+            for root, dirs, local_files in os.walk(local_folder):
+                key = root
+                annotation_file = local_files
```

Comment:
`local_files` is a list, not a single annotation_file.

Reply:
Changed the logic to handle multiple files